js/service/shell/index.js

```diff
@@ -45,3 +46,19 @@ exports.setCommand = function(name, cb) {
   assert(typeutils.isFunction(cb));
   commands.set(name, cb);
 };
+
+exports.runCommand = function(name, args) {
```

Comment:
Yes! And it should be named `env` in the callbacks in order to override the global `env` i'm adding.